jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/python/PythonLambdaHandlerResolverTest.kt

```diff
@@ -140,6 +141,20 @@ class PythonLambdaHandlerResolverTest {
         assertHandler("app.handle", true)
     }
 
+    @Test
+    fun pyTestHandlersIgnored() {
+        createPyTestHandler("hello_world/app.py")
+
+        assertHandler("doesnt_exist", false)
+    }
+
+    @Test
+    fun handlesInDirectoriesMarkedTestNotFound() {
+        val vfs = createHandler("tst/app.py")
+        PsiTestUtil.addSourceRoot(projectRule.module, vfs.parent, true)
+        assertHandler("doesnt_exist", false)
```

Comment:
of course it doesnt exist, this method isnt in the class!

Reply:
Ugh, fixed